lib/elixir/lib/access.ex

```diff
@@ -195,8 +195,10 @@ defmodule Access do
       :error -> nil
     end
 
-    {get, update} = fun.(current_value)
-    {get, :maps.put(key, update, map)}
+    case fun.(current_value) do
+      {:skip, _} -> {:ok, map}
```

Comment:
This change unfortunately break a code that would call get_and_update_in and return `:skip` as the "gotten" value. Ideally, we should make this work without this change here.